docs/getting-started/integrating-codacy-with-your-git-workflow.md

```diff
@@ -22,7 +22,7 @@ To integrate Codacy with your Git workflow, follow these steps:
 [Review and adjust the quality gates](../repositories-configure/adjusting-quality-gates.md) of your repository to decide which pull requests should fail the Codacy quality gate.
 
 !!! tip
-    The default values of the quality gate rules are designed to help maintain the current code quality of your repository. In particular, the default value for the coverage rule might be demanding. Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
+    The default quality gate rules defined by [the built-in **Codacy gate policy**](../organizations/using-gate-policies.md) are designed to help maintain the current code quality of your repository. In particular, the default value for the coverage rule might be demanding. Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
```

Comment:
Maybe we don't need or want to provide so much detail on the tutorial. Users may not yet know what quality gate rules are yet, much less what a "built-in Codacy gate policy" is. :sweat_smile: It seems like it's a lot of concepts to take in all at once. Maybe there's also other ways to simplify.

```suggestion
    The [default quality gate rules]](../organizations/using-gate-policies.md) are designed to help maintain the current code quality of your repository. In particular, the default value for the coverage rule might be demanding. Depending on factors such as the current code quality of your repository and the maturity of your team practices, consider the balance between implementing stricter quality gates and the possibility of delaying or blocking the development progress.
```

Reply:
I agree, the reader doesn't need this level of detail at this point. 👍